internal/clients/connect/client.go

```diff
@@ -30,5 +30,5 @@ type APIClient interface {
 	DeployBundle(types.ContentID, types.BundleID, logging.Logger) (types.TaskID, error)
 	WaitForTask(taskID types.TaskID, log logging.Logger) error
 	ValidateDeployment(types.ContentID, logging.Logger) error
-	CheckCapabilities(util.AbsolutePath, *config.Config, logging.Logger) error
+	CheckCapabilities(util.AbsolutePath, *config.Config, *types.ContentID, logging.Logger) error
```

Comment:
If you didn't notice it in the previous file, I've had to add a pointer to an optional contentID into this interface.